pkg/noderesourcetopology/README.md

```diff
@@ -262,22 +262,22 @@ zones:
          $ kubectl get noderesourcetopologies.topology.node.k8s.io
        ```
 
-    1. Alternatively, in case you are just interested in simply testing the scheduler plugin, use the manifest in the manifest directory to deploy the CRD and CRs as follows:
+    2. Alternatively, in case you are just interested in simply testing the scheduler plugin, use the manifest in the manifest directory to deploy the CRD and CRs as follows:
```

Comment:
markdown renderer should automatically convert the list index without the need for humans to enforce the correct ordering: https://www.markdownguide.org/basic-syntax/#ordered-lists

Reply:
got it. done